.scalafmt.conf

```diff
@@ -0,0 +1,45 @@
+style = defaultWithAlign
```

Comment:
We usually make a commit with scalafmt config and the project after being formatted.  Then the PR is based on the already formatted project so it is easier to tell the code changes from formatting changes.

Reply:
I didn't know it, sorry.